server/src/main/java/org/kaazing/gateway/server/context/resolve/GatewayContextResolver.java

```diff
@@ -1246,6 +1253,11 @@ private void injectResources(Collection<ServiceContext> services,
         // inject bridge service factory
         injectResources(bridgeServiceFactory, injectables);
 
+        // inject websocket extensions
+        for (WebSocketExtensionFactorySpi factory : webSocketExtensionFactory.availableExtensions()) {
```

Comment:
Do we want gateway.server need to know about WebSocketExtensionFactorySpi ?  Can it not be added on WsTransport/WsnTransport  etch that return extension instances. Say, List<WebSocketExtension> WsTransport#getExtensions(...).

Just thinking aloud (I need to go through the rest of changeset).